concepticondata/concepticon.tsv

```diff
@@ -2672,3 +2672,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2671	PENIS SHEATH	Clothing and grooming	Sheath to cover a penis	Person/Thing	
 2672	HEADWATERS	The physical world	The source of a river or stream	Person/Thing	
 2673	BIRD OF PARADISE	Animals	A bird of the family Paradisaeidae.	Person/Thing	
+2674	ORAL CAVITY	The body	Cavity that is limited by the lips in front, the cheeks on the side, the soft and hard palate at the top and the floor of the mouth below.	Person/Thing	
+2675	MAN OR MALE	Kinship	Relating to the male sex of humans.	Property	
```

Comment:
ok, I'll remove those two (MAN OR MALE and WOMAN OR FEMALE). No information in Stokhof about what they actually are.